language-server/modules/langserver-compiler/src/main/java/org/ballerinalang/langserver/compiler/workspace/repository/LSPathConverter.java

```diff
@@ -30,15 +30,17 @@
  */
 public class LSPathConverter extends PathConverter {
     private WorkspaceDocumentManager documentManager;
+    private final Path root;
     
     public LSPathConverter(Path root, WorkspaceDocumentManager documentManager) {
         super(root);
+        this.root = root;
```

Comment:
Super class `PathConverter` already have a field called root. Re-use that. 

Super constructor invocation `super(root)` will assign the value to that variable. Hence we don't need to explicitly assign either.

Reply:
Ack; Addressed by commit 0c9d4fd.